solidity/dashboard/src/components/SideMenu.jsx

```diff
@@ -1,35 +1,72 @@
 import React from "react"
-import { Link, useRouteMatch } from "react-router-dom"
+import { Link, NavLink, useRouteMatch } from "react-router-dom"
 import * as Icons from "./Icons"
+import { isEmptyArray } from "../utils/array.utils"
+import Divider from "./Divider"
+
+const styles = {
+  overviewDivider: { margin: "1rem 1.5rem" },
+}
 
 export const SideMenu = (props) => {
   return (
     <nav className="side-menu--active">
       <ul className="side-menu__list">
-        <NavLink to="/overview" label="overview" icon={<Icons.Home />} />
-        <NavLink
-          to="/delegation"
-          label="delegation"
-          icon={<Icons.FeesVector />}
+        <li className="side-menu__route-wrapper">
+          <NavLink
+            to="/overview"
+            className="side-menu__route"
+            activeClassName="side-menu__route--active"
+          >
+            Overview
+          </NavLink>
+          <Divider style={styles.overviewDivider} />
+        </li>
+        <NavLinkSection
+          label="stake"
+          icon={<Icons.StakeDrop />}
+          subroutes={[
+            {
+              label: "Delegation",
+              path: "/delegation",
```

Comment:
This name sits a little weird. It seems categorical, unlike the rest of the links (grants, applications, operations, etc). Perhaps “Delegations” would work? Since we also include information about other delegation activities.

Not really willing to die on this hill, just feels a little odd as is. cc @liz-shinn perhaps.

Reply:
Yup, I also think that "Delegations" is a good change. Makes sense to me @Shadowfiend